e2e/main.go

```diff
@@ -177,14 +177,15 @@ func (d *dockerManager) StartContainers() error {
 }
 
 func (d *dockerManager) StopContainers() error {
-	if strings.Contains(d.dir, "dockerv2") {
-		_, err := exec.Command("docker",
+	if strings.Contains(d.dir, "path") {
```

Comment:
Why the specific check here?

Reply:
Oops. Leftover from debugging. Will remove it :)

Reply:
Debugging Leftover. :sweat_smile: